pages/_app.js

```diff
@@ -4,11 +4,13 @@ import { initialCountries } from "@/lib/db";
 import useLocalStorageState from "use-local-storage-state";
 import NavBar from "@/components/NavBar";
 import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
   const [countries, setCountry] = useLocalStorageState("countries", {
     defaultValue: initialCountries,
   });
+  const [isCurrentCountry, setIsCurrentCountry] = useLocalStorageState();
```

Comment:
The name is misleading.

`isSomething` denotes a Boolean type. For example,
- isTrue
- isEmpty
- isAvailable

But your state is not a Boolean. It's an object if I'm not mistaken. Give it a proper name to reflect what it actually is.